#163392083 create a loginPage

vkingmaker opened the pull request:
#### What does this PR do?
- Add a login page to the project to from where users can login to use the application
#### Description of Task to be completed?
`/GET /login.html`
#### How should this be manually tested?
- After cloning the repository
- `cd` into the project.
- run a local server to host the app locally. 
- using your favorite browser visit the link above.
#### What are the relevant pivotal tracker stories?
#163392083
#### Screenshots
![loginup](https://user-images.githubusercontent.com/18199741/51687940-d1453b00-1ff3-11e9-8e2b-d88924a4c460.PNG)

vkingmaker committed:
feature(loginPage): create a loginPage

This commit will add a login page to enable user login to be able to use the core features of the application

[Delivers #163392083]

megame24 commented:
The text in the blue button is not visible, I suggest you find another color for the text. Also, I suggest you add border-radius to your buttons.

vkingmaker committed:
feature(loginPage): create a loginPage

This commit will add a login page to enable user login to be able to use the core features of the application

[Delivers #163392083]

megame24 reviewed (approved):
Looks good, you can merge
